OpenSource/checklists/01-set-up.md

```diff
@@ -47,13 +47,12 @@
 
 - [ ] Repository includes a complete developer documentation
 - [ ] Developer documentation includes all installation steps, including prerequisites (with link to the specific part in the tool documentation whenever needed)
-- [ ] Anyone, on any operating system, can fulfill the prerequisites for development (dev and testing tools) within an hour
```

Comment:
@m4rk055 I see we already discussed this in https://github.com/blindnet-io/openness-framework/pull/21#discussion_r860917356. Would it be ok for you to add the missing items?

Making sure "anyone" can contribute to our OSS projects is a top priority.

Reply:
They were [added](https://github.com/blindnet-io/openness-framework/pull/21/files#diff-82e845cd95e71d960501c57661279e4ff8f5e8277ec65d398af5c8549df78a6fR59). I just forgot to remove this point.

Reply:
Oh OK, my bad :sweat_smile: